Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt

```diff
@@ -200,19 +200,43 @@ class VungleBannerAdapter(adapter: VungleAdapter) :
         }
     }
 
-    private fun getBannerSize(bannerSize: ISBannerSize): BannerAdSize? {
-        return when (bannerSize.description) {
-            "BANNER", "LARGE" -> BannerAdSize.BANNER
-            "RECTANGLE" -> BannerAdSize.VUNGLE_MREC
+    private fun getBannerSize(bannerSize: ISBannerSize): VungleAdSize? {
+        val vungleAdSize = when (bannerSize.description) {
+            "BANNER", "LARGE" -> VungleAdSize.BANNER
+            "RECTANGLE" -> VungleAdSize.MREC
             "SMART" ->
                 (if (AdapterUtils.isLargeScreen(ContextProvider.getInstance().applicationContext)) {
-                    BannerAdSize.BANNER_LEADERBOARD
+                    VungleAdSize.BANNER_LEADERBOARD
                 } else {
-                    BannerAdSize.BANNER
+                    VungleAdSize.BANNER
                 })
 
+            "CUSTOM" -> VungleAdSize.getAdSizeWithWidthAndHeight(
+                bannerSize.width,
+                bannerSize.height
+            )
+
             else -> null
         }
+
+        try {
+            if (bannerSize.isAdaptive && vungleAdSize != null) {
```

Comment:
From IS's integration guide, they recommend to set in their integration guide.
```
val bannerSize = ISBannerSize.BANNER// the banner size you choose will be the default value for non supporting banner adaptive networks
```
So, there should be a case in which `isAdaptive == true` and `vungleAdSize = VungleAdSize.BANNER` at this line?






https://developers.is.com/ironsource-mobile/android/banner-integration-android/#step-6